lua/conform/formatters/ruff.lua

```diff
@@ -15,5 +15,6 @@ return {
   stdin = true,
   cwd = require("conform.util").root_file({
     "pyproject.toml",
+    "ruff.conf",
```

Comment:
Sorry there was a typo here, it should be `ruff.toml`, same in the other formatted.